package.json

```diff
@@ -49,7 +49,10 @@
     "react-test-renderer": "^16.13.1"
   },
   "jest": {
-    "preset": "react-native"
+		"preset": "react-native",
```

Comment:
There seems to be some funny whitespace here, maybe it's the attack of the tabs again.

Reply:
I need to change something in my vscode settings...